Cannot compile 2.27.0 with `wrap_help` feature - unresolved import `term_size`

<!--
Please use the following template to assist with creating an issue and to ensure a speedy resolution. If an area is not applicable, feel free to delete the area or mark with `N/A`
-->

Starting a blank project with clap 2.27.0 as a dependency, and enabling the `wrap_help` feature fails to compile.

### Rust Version

* rustc 1.21.0 (3b72af97e 2017-10-09)

### Affected Version of clap

* clap 2.27.0

### Expected Behavior Summary

Compiles successfully.

### Actual Behavior Summary

Fails to compile.

### Steps to Reproduce the issue

1.  Create a new binary project: `cargo init --bin`
2.  Add the following lines to Cargo.toml:
```toml
[dependencies.clap]
version = "2.27"
features = ["wrap_help"]
```
3.  Run `cargo build`
4. Observe the following error:
```
   Compiling clap v2.27.0
error[E0432]: unresolved import `term_size`
  --> /Users/username/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-2.27.0/src/app/help.rs:22:5
   |
22 | use term_size;
   |     ^^^^^^^^^ no `term_size` in the root

error: aborting due to previous error

error: Could not compile `clap`.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,11 @@
+<a name="v2.27.1"></a>
+## v2.27.1 (2017-10-24)
+
+
+#### Bug Fixes
+
+* Adds `term_size` as an optional dependency (with feature `wrap_help`) to fix compile bug
+
 <a name="v2.27.0"></a>
 ## v2.27.0 (2017-10-24)
 
@@ -6,7 +14,7 @@
 ** This release also contains a very minor breaking change to fix a bug **
 
 The only CLIs affected will be those using unrestrained multiple values and subcommands where the
-subcommand name can coincide with one of the multiple values. 
+subcommand name can coincide with one of the multiple values.
 
 See the commit [0c223f54](https://github.com/kbknapp/clap-rs/commit/0c223f54ed46da406bc8b43a5806e0b227863b31) for full details.
 

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 
 name = "clap"
-version = "2.27.0"
+version = "2.27.1"
 authors = ["Kevin K. <kbknapp@gmail.com>"]
 exclude = ["examples/*", "clap-test/*", "tests/*", "benches/*", "*.png", "clap-perf/*", "*.dot"]
 repository = "https://github.com/kbknapp/clap-rs.git"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -29,6 +29,7 @@ yaml-rust = { version = "0.3.5",  optional = true }
 clippy    = { version = "~0.0.166", optional = true }
 atty      = { version = "0.2.2",  optional = true }
 vec_map   = { version = "0.8", optional = true }
+term_size = { version = "0.3.0", optional = true }
 
 [dev-dependencies]
 regex = "0.2"
@@ -39,7 +40,7 @@ version-sync = "0.3"
 default     = ["suggestions", "color", "vec_map"]
 suggestions = ["strsim"]
 color       = ["ansi_term", "atty"]
-wrap_help   = ["textwrap/term_size"]
+wrap_help   = ["term_size", "textwrap/term_size"]
 yaml        = ["yaml-rust"]
 unstable    = [] # for building with unstable clap features (doesn't require nightly Rust) (currently none)
 nightly     = [] # for building with unstable Rust features (currently none)

--- a/README.md
+++ b/README.md
@@ -45,7 +45,7 @@ Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
 
 ## What's New
 
-Here's whats new in 2.27.0:
+Here's whats new in 2.27.1:
 
 ** This release also contains a very minor breaking change to fix a bug **
 
@@ -54,6 +54,7 @@ subcommand name can coincide with one of the multiple values.
 
 See the commit [0c223f54](https://github.com/kbknapp/clap-rs/commit/0c223f54ed46da406bc8b43a5806e0b227863b31) for full details.
 
+*    Adds `term_size` as an optional dependency (with feature `wrap_help`) to fix compile bug
 *   **The minimum required version of Rust is now 1.18.0 (Stable)**
 *   Values from global args are now propagated UP and DOWN!
 *   fixes a bug where using AppSettings::AllowHyphenValues would allow invalid arguments even when there is no way for them to be valid

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -513,7 +513,7 @@
 //! this repository for more information.
 
 #![crate_type= "lib"]
-#![doc(html_root_url = "https://docs.rs/clap/2.27.0")]
+#![doc(html_root_url = "https://docs.rs/clap/2.27.1")]
 #![deny(
         missing_docs,
         missing_debug_implementations,

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -544,6 +544,8 @@ extern crate unicode_width;
 extern crate bitflags;
 #[cfg(feature = "vec_map")]
 extern crate vec_map;
+#[cfg(feature = "wrap_help")]
+extern crate term_size;
 extern crate textwrap;
 #[cfg(feature = "color")]
 extern crate atty;

